docs/wiki/research/SSF.md

```diff
@@ -0,0 +1,57 @@
+# Single slot finality (SSF)
+
+## Roadmap tracker
+
+| Upgrade |   URGE    | Track |          Topic           |                                               Cross-references                                               |
+| :-----: | :-------: | :---: | :----------------------: | :----------------------------------------------------------------------------------------------------------: |
+|   SSF   | the Merge |   -   | Increase Validator count | intersection with: [MAX_EB](/docs/wiki/research/cl-upgrades.md), [ePBS](/docs/wiki/research/PBS/ePBS.md), ET |
```

Comment:
Checking Vs picture again. It seems that SSF *is* part of the increasing validator count track. No topic tho, since there's only one topic and that overlaps with the track (in contrast to other urges e.g. scourge where there are multiple topics spread over 2 tracks).
So, I propose 
- track : increasing validator count
- topic : - 

Regarding intersections: SSLE seems legit. But I would keep max_eb (closely related to enabling greater validator count), and yes PBS

Reply:
The overall track here should be something like PoS Improvements and topic more specific, something like faster finalization/economic security. I don't want to leave out empty columns here, otherwise I would make them optional or remove in the template for these tables.

The picture from V shouldn't be an ultimate reference tool because it's changing and doesn't reflect all realities. 
SSF makes it harder to increase validator count, the more validators, the harder the aggregation in one slot. The idea in the picture is to build on top of this to make it more effective.